specs/toolsets/php_8/setup-toolset.sh

```diff
@@ -29,7 +29,7 @@ case $HAM_OS in
         set -ex
         sudo add-apt-repository -y ppa:ondrej/php
         sudo apt-get -y update
-        ham-apt-get-install php$HAM_PHP_VERSION-common php$HAM_PHP_VERSION-mysql php$HAM_PHP_VERSION-xml php$HAM_PHP_VERSION-xmlrpc php$HAM_PHP_VERSION-curl php$HAM_PHP_VERSION-gd php$HAM_PHP_VERSION-imagick php$HAM_PHP_VERSION-cli php$HAM_PHP_VERSION-dev php$HAM_PHP_VERSION-imap php$HAM_PHP_VERSION-mbstring php$HAM_PHP_VERSION-opcache php$HAM_PHP_VERSION-pgsql php$HAM_PHP_VERSION-soap php$HAM_PHP_VERSION-sqlite3 php$HAM_PHP_VERSION-zip php$HAM_PHP_VERSION-intl php$HAM_PHP_VERSION-fpm php$HAM_PHP_VERSION-gmp
```

Comment:
Why all this extensions are added only in the linux version? Should I do something with the macos and win versions too?

Reply:
On macOS and Windows those are all built in / packaged in, for some reason on Linux making a package per tiny extension seems worthwhile to them... Maybe we should just look at the php.ini for windows and just install all of those on Linux. So that stop playing whackamole with them, there isnt much value in saving a few MB by not including all of them imo.

cf: https://stackoverflow.com/questions/60105096/how-to-install-bcmath-in-php-7-3-on-windows